src/path.cr

```diff
@@ -540,6 +540,18 @@ struct Path
     end
   end
 
+  # Converts this path to an absolute path. Relative paths are
+  # referenced from the current working directory of the process (`Dir.current`)
+  # unless *base* is given, in which case it will be used as the reference path.
```

Comment:
There doesn't seem to be enough difference in the first paragraph of these doc comments. You should probably add the 

>"~/" is expanded to the current user's home directory.

in `#expand`'s docs

Reply:
`#expand` contains documentation on when and what `home` is expanded to.